src/SpacetimeDBClient.cs

```diff
@@ -64,15 +64,15 @@ public DbConnectionBuilder<DbConnection, Reducer> OnConnect(Action<DbConnection,
             return this;
         }
 
-        public DbConnectionBuilder<DbConnection, Reducer> OnConnectError(Action<WebSocketError?, string> cb)
+        public DbConnectionBuilder<DbConnection, Reducer> OnConnectError(Action<Exception> cb)
         {
-            conn.webSocket.OnConnectError += (a, b) => cb.Invoke(a, b);
+            conn.webSocket.OnConnectError += (e) => cb.Invoke(e);
             return this;
         }
 
-        public DbConnectionBuilder<DbConnection, Reducer> OnDisconnect(Action<DbConnection, WebSocketCloseStatus?, WebSocketError?> cb)
+        public DbConnectionBuilder<DbConnection, Reducer> OnDisconnect(Action<DbConnection, Exception?> cb)
```

Comment:
Same here, how about adding a string to the event signature?

Because these callbacks serve two purposes; offer an error message, and offer a dispatch mechanism. With Exception wrapping string, the latter is lost, no?

Reply:
Same as above